torchgeo/trainers/detection.py

```diff
@@ -50,7 +50,7 @@ class ObjectDetectionTask(pl.LightningModule):
     def config_task(self) -> None:
```

Comment:
Did you want to add the same blurb above saying where to find a list of available models and backbones?

Reply:
yes, that would be good a good documentation.  don't know what the best way is in this case, but I made a proposal